OrbitGl/ThreadTrack.cpp

```diff
@@ -157,8 +159,10 @@ void ThreadTrack::Draw(GlCanvas* canvas, PickingMode picking_mode) {
 void ThreadTrack::OnPick(int /*x*/, int /*y*/) { GOrbitApp->set_selected_thread_id(thread_id_); }
 
 void ThreadTrack::UpdatePrimitives(uint64_t min_tick, uint64_t max_tick, PickingMode picking_mode) {
-  event_track_->SetPos(pos_[0], pos_[1]);
-  event_track_->UpdatePrimitives(min_tick, max_tick, picking_mode);
+  if (thread_id_ != SamplingProfiler::kAllTracepointsFakeTid) {
+    event_track_->SetPos(pos_[0], pos_[1]);
+    event_track_->UpdatePrimitives(min_tick, max_tick, picking_mode);
+  }
```

Comment:
Same as above, it would be better to avoid introducing a dependency to SamplingProfiler.